cardano-api/test/Test/Cardano/Api/Json.hs

```diff
@@ -3,6 +3,8 @@
 {-# LANGUAGE LambdaCase #-}
 {-# LANGUAGE TemplateHaskell #-}
 
+{-# OPTIONS_GHC -Wno-deprecations #-} -- TODO Fix deprecations
```

Comment:
What deprecations?

Reply:
Not sure why this appeared in your review, but it's not in the PR.